### PR TITLE
Fix playlist manifest IIIF validator errors

### PR DESCRIPTION
**Pull request by Dananji**
Related issue: #5802

Related `iiif_manifest` gem PR: https://github.com/samvera/iiif_manifest/pull/112

### CODE REVIEW

File: app/models/iiif_playlist_canvas_presenter.rb [hunk 1]
```diff
@@ -218,6 +209,16 @@ def manifest_attributes(quality, media_type)
...
     end
 
+    def placeholder_attributes(label_content)
+      placeholder_hash = {
+        label: label_content,
+        type: 'Text',
+        format: 'text/plain',
+        width: 1280,
+        height: 720,
+    }.compact
```
Comment (masaball) [inline app/models/iiif_playlist_canvas_presenter.rb:219]: Not a blocker, but looks like this line didn't get indented properly.

**Review by masaball — COMMENTED** [review]
Do we need to include pulling in the iiif_manifest update in this PR as well? Or will it be fine to pull that in as a separate PR later?

**Comment by Dananji** [comment]
I updated the `Gemfile` to fetch `iiif_manifest` from Git repo. Once the changes are are tagged and released we can update the `Gemfile`?

**Review by masaball — APPROVED** [review]
Yeah, we can pull from the git repo for testing in develop. We'll tag a iiif_manifest release and update the `Gemfile` to point to that when we are finalizing the next Avalon release.

**Review by masaball — APPROVED** [review]

**Pull request merged**